docs/features/feature-guides/compliance-forms/complete-a-form.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Complete a Form
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# Complete a DataHub Compliance Form
+<FeatureAvailability/>
+
+This guide provides an example of completing a Compliance Form, covering:
```

Comment:
How about we just start with..

Once a Compliance Form has been published (link to the Creating a Compliance Form doc), assignees will be notified via their **Inbox** to complete the Form for each asset they are responsible for.